.gitignore

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
do you need all of those now?

Reply:
I removed anything related with the ```*_old``` files.

Reply:
do you need those too
```
test/Testing/Temporary/LastTest.log
test/Testing/Temporary/CTestCostData.txt
```
are the created by cython or something?

Reply:
These were there in the initial ```.gitignore``` file. 
Would you like to remove them?

Reply:
Yes, if you check that those files are not generated locally on your side, which is my guess.